### DISCUSSION TITLE
Search item inside a menu

### DISCUSSION THREAD

**Discussion by cubecleveland** [tailwindlabs/headlessui#684] [Help]
Hello, im trying to create a dropdown menu that consists on menu items that are renderd from ajax call. they come in paginated so i added a search box inside the menu an an item to search and fire an ajax call.
when the ajax call comes back i redraw the menu...

All works well but i feel im doing it wrong or is there another way to do it...
the main bug we have now is that if u press the space bar inside the search box the whole menu closes....

Hope i can get some help.

**Comment by ADTC** [comment]
See #820
